src/base/ApplicationDynamoDBTable.ts

```diff
@@ -140,6 +146,52 @@ export class ApplicationDynamoDBTable extends Resource {
       ],
       dependsOn: [targetTracking, dynamoDB],
     });
+
+    // TODO: do we want to assume that all global secondary indexes get a scaling policy?
+    if (globalSecondaryIndexes.length) {
+      globalSecondaryIndexes.forEach((gsIndex) => {
+        // min capacity is defined by the global secondary index
+        // max capacity is inherited from the table auto scaling config
+        // TODO: is this a good forced/default behavior?
```

Comment:
I think it should be the default behavior but not forced.

Reply:
so the question becomes - do we extend the `DynamodbTableGlobalSecondaryIndex` interface to include a `maxCapacity` property? i'm not a huge fan of extending third-party indexes, but this does seem to be a gap in functionality/configurability.

(note that the `DynamodbTableGlobalSecondaryIndex` interface defines a `writeCapacity` and `readCapacity` - which i think makes sense to use as the `minCapacity` - but happy to hear other thoughts if we need to extend this interface with both `minCapacity` and `maxCapacity`.)